src/components/PipelineRunDetailsView/tabs/PipelineRunDetailsTab.tsx

```diff
@@ -122,6 +134,14 @@ const PipelineRunDetailsTab: React.FC<React.PropsWithChildren<PipelineRunDetails
                   <DescriptionListTerm>Duration</DescriptionListTerm>
                   <DescriptionListDescription>{duration ?? '-'}</DescriptionListDescription>
                 </DescriptionListGroup>
+                {snapshotCreationStatus && (
+                  <DescriptionListGroup>
+                    <DescriptionListTerm>Snapshot creation status</DescriptionListTerm>
+                    <DescriptionListDescription>
+                      <Truncate content={snapshotCreationStatus.message} />
```

Comment:
I think we should not truncate the message because it could contain important information. We should show the status in a different way maybe using success and failure icon and then show the message in a separate block similar to how PLR failure message is shown.

Reply:
Removed the truncating of message. 
We discussed here that initially, we would only display the message as is for all statuses, [slack thread](https://redhat-internal.slack.com/archives/C038DJAP7HR/p1718814065143589)

If needed we can later have a story with detailed UX designs as the message internal structure for various statuses is different.